prow/plugins/cla/cla.go

```diff
@@ -88,12 +88,14 @@ func handle(gc gitHubClient, log *logrus.Entry, se github.StatusEvent) error {
 
 	org := se.Repo.Owner.Login
 	repo := se.Repo.Name
+	log.Info("Searching for PRs matching the commit.")
 	issues, err := gc.FindIssues(url.QueryEscape(fmt.Sprintf("%s repo:%s/%s type:pr state:open", se.SHA, org, repo)))
 	if err != nil {
```

Comment:
I would say we should log the PR#s found as well, if it has more than 1. What do you think @spxtr?